### DISCUSSION TITLE
Crag Gardens should use more blocks to detail their walls.

### DISCUSSION THREAD

**Discussion by TarotFooling** [CorgiTaco/BYG#1185] [Ideas]
Gonna be throwing out the format for this one, since it's such a simple request.
The walls of the crag_gradens could be spiced up with blocks such as stone, mossy cobblestone, cobblestone, and dirt.
The current wall of repeating textures looks somewhat comical, and really looks out of place in the beauty of the mod. 
[https://i.imgur.com/REJMA8e.png](https://i.imgur.com/REJMA8e.png)

**Comment by doppeltrouble** [comment]
like the idea
but i would rather have the 1.20.1 update than reduce the dev's time with changing color palettes and texture of an extremely rare biome. and if you use terralith you can look for the jungle mountains biome which is similar and if you dont like all the biomes from terralith use biome replacer to change that. 
https://www.curseforge.com/minecraft/mc-mods/biome-replacer

**Reply by doppeltrouble** [reply]
sorry that looks more rude than i wanted it to be.

**Reply by dairymoose** [reply]
What makes me sad is there's a twitter post of them getting BYG working in 1.20.1 like 1 year ago.  I wish we could just have that while we wait for the big mega update that's coming.  My world feels small with the vanilla biomes

**Reply by doppeltrouble** [reply]
yeah i agree but i see this new discussion where they talk about Alpha release on patreon and then Beta so im defintely going to check that out!!!

**Reply by doppeltrouble** [reply]
Also i only really use BYG or BWG as its called now for like 7 or 8 biomes and i use the biome swapper to get rid of the rest so as long as they are coming im going to accept everything they do with this mod

**Comment by suerion** [comment]
BWG and BYG are not compatible, i read that in the discord often, 

BYG will only Support to 1.19.4, BWG is a Mod that would be fit in 1.20.1 and above.

on Fabric Discord i had written for an Compat Mod to use the Old BYG with BWG

https://discord.com/channels/507304429255393322/1240238900383973407

**Reply by doppeltrouble** [reply]
cool but i dont have old BYG worlds so i really only need a 1.20.1 version (forge) that adds the needed content for my modpack. also love the work people (not me, im too stupid) put in for my favorite mods.

**Reply by suerion** [reply]
Then you should waut for BWG, BYG will not support 1.20.1 or above

**Reply by doppeltrouble** [reply]
that is what i was trying to say so... thanks.

**Reply by doppeltrouble** [reply]
waaaiit the compat mod implies that you have access to al least an alpha version of BWG is it already out on patreon?

**Reply by doppeltrouble** [reply]
*at least not al least and . after BWG
